src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Lottery {
+    private final int randomNumber = (int) (Math.random() * 100);
```

Comment:
1. it's not actually random value in your solution - it'll be calculated once and every time we'd get identic color
2. magic number

Reply:
updated